src/main/scala/io/gitlab/agluszak/tickets/Server.scala

```diff
@@ -0,0 +1,52 @@
+package io.gitlab.agluszak.tickets
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import io.gitlab.agluszak.tickets.PostgresProfile.api._
+import io.gitlab.agluszak.tickets.controllers.{ReservationController, ScreeningController}
+import io.gitlab.agluszak.tickets.repositories.{ReservationRepository, ScreeningRepository, TicketTypeRepository}
+import io.gitlab.agluszak.tickets.services.{ReservationService, ScreeningService}
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+class Server(implicit system: ActorSystem, materializer: ActorMaterializer, executor: ExecutionContextExecutor) {
+  val db: Database = Database.forConfig("postgres")
```

Comment:
Do you think that "parsing" configuration here is a good idea? If so why?

Reply:
Because it's simple. I could introduce a dedicated service or use simply use pureconfig